### DISCUSSION TITLE
Student Developer Pack benefits

### DISCUSSION THREAD

**Discussion by LordSaumya** [community/community#136521] [GitHub Education]
### Hi everyone,

Hi!

My request for the student developer pack was approved three days ago, but I haven't received my benefits yet.

**Comment by ferid333** [comment]
It’s great that your request was approved! Sometimes it can take a few days for the benefits to become available. If you still haven’t received them, try checking your email for any instructions or updates. If nothing shows up, consider reaching out to GitHub support for help. 🤗

**Comment by davevad93** [comment]
Hi @LordSaumya ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @LordSaumya was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; 

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
